sdk/python/endpoints/online/mlflow/sklearn-diabetes/environment/conda.yml

```diff
@@ -0,0 +1,13 @@
+channels:
+- conda-forge
+dependencies:
+- python=3.7.11
```

Comment:
not that important for this PR, but we should probably start upgrading examples to 3.8 soon

Reply:
Indeed Walter, I already started in another PR!